src/nodes/accessors/InstanceNode.js

```diff
@@ -229,7 +229,7 @@ class InstanceNode extends Node {
 
 			// update version if necessary
 
-			if ( this.instanceMatrix.usage !== DynamicDrawUsage && this.instanceMatrix.version !== this.buffer.version ) {
+			if ( this.instanceMatrix.version !== this.buffer.version ) {
```

Comment:
These `DynamicDrawUsage` checks are redundant. What matters is to set `DynamicDrawUsage` at buffer attribute node level.